templates/caddy.conf

```diff
@@ -27,5 +28,5 @@ limit nofile 1048576 1048576
 script
         cd /etc/caddy
```

Comment:
Based on https://caddyserver.com/docs/automatic-https it is supposed to be CADDYPATH, so it is correct this commit did not change it.